stock_custom_exchange_rate_date/tests/test_stock.py

```diff
@@ -0,0 +1,154 @@
+from datetime import timedelta
+
+from odoo import fields
+from odoo.tests import Form, TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestStock(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.vendor = cls.env.ref("base.res_partner_1")
+        cls.usd = cls.env.ref("base.USD")
+        cls.eur = cls.env.ref("base.EUR")
+        cls.today = fields.Date.context_today(cls.env.user)
+        cls.yesterday = cls.today - timedelta(days=1)
+        account = cls.env["account.account"].create(
+            {"name": "Receivable", "code": "RCV00", "account_type": "asset_receivable", "reconcile": True}
```

Comment:
Could you use one per line, please? For readability (add a trailing comma)

Same below.